src/OrbitQt/HistogramWidget.cpp

```diff
@@ -279,7 +317,7 @@ void HistogramWidget::paintEvent(QPaintEvent* /*event*/) {
 
   const orbit_statistics::Histogram& histogram = histogram_stack_.top();
 
-  QPoint axes_intersection(kLeftMargin, Height() - kBottomMargin);
+  const QPoint axes_intersection(kLeftMargin, Height() - kBottomMargin);
```

Comment:
Try to make unrelated cleaning changes in a different PR to facilitate review process.